Alloy/commands/compile/parsers/Ti.UI.Android.CollapseToolbar._ProxyProperty.js

```diff
@@ -0,0 +1,12 @@
+var _ = require('lodash');
+
+exports.parse = function(node, state) {
+	_.extend(state, {
```

Comment:
I think it'd be worthwhile guarding this against usage in unsupported SDKs, there's an example in [Ti.UI.OptionBar](https://github.com/tidev/alloy/blob/792ba0fa74e5a32d1c4ee2a6cce3ece30728c70d/Alloy/commands/compile/parsers/Ti.UI.OptionBar.js#L7-L10C3), I _think_ it'd look like as follows

```
	const MIN_VERSION = '12.1.0';
	const tiappSdkVersion = tiapp.getSdkVersion();
	if (tiapp.version.lt(tiappSdkVersion, MIN_VERSION)) {
		U.die(`Ti.UI.CollapseToolbar requires Titanium SDK ${MIN_VERSION}+`);
	}
```